usr/libexec/greenboot/greenboot

```diff
@@ -82,7 +84,10 @@ case "$1" in
   "check")
     rc=0
     for health_check_path in "${SCRIPTS_CHECK_PATHS[@]}"; do
-      script_runner "$health_check_path/required.d" "strict" "Running Required Health Check Scripts..." || rc=1
+      script_runner "$health_check_path/required.d" "strict" "Running Required Health Check Scripts..." || {
+        rc=1
+        break
+      }
       script_runner "$health_check_path/wanted.d" "relaxed" "Running Wanted Health Check Scripts..."
     done
     print_unexecuted_checks
```

Comment:
@djach7  do you remember why printing the disabled scripts again when it's already printed in script_runner()?

Reply:
I don't remember, no. That doesn't make much sense to me reading it now.

Reply:
remove scripts that are irrelevant.